postgres/datadog_checks/postgres/statements.py

```diff
@@ -152,6 +160,26 @@ def _get_pg_stat_statements_columns(self):
         col_names = [desc[0] for desc in cursor.description] if cursor.description else []
         self._stat_column_cache = col_names
         return col_names
+    
+    def _track_io_timing_enabled(self):
```

Comment:
Instead of writing a new query + cache for this specific setting, better add it to the list of settings we're already reading out in in `postgres.py:_load_pg_settings`